src/graphql/resolvers/state-resolver.ts

```diff
@@ -34,4 +42,38 @@ export class StateResolver {
   async params(@Ctx() context: GraphQLContext, @Info() info: GraphQLResolveInfo) {
     return await context.repository.epochs.getLast(info);
   }
+
+  @FieldResolver()
+  async network() {
+    try {
+      const response = await this._nodeService.getNodeInfo();
+      return response.data["network"];
+    } catch (e) {
+      if (e instanceof AxiosError) {
+        if (e.response?.data?.error === 400) {
+          throw new GraphQLError(e.response?.data?.detail);
+        }
+      }
+
+      console.error(e);
+      throw new GraphQLError("Unknown error");
+    }
+  }
+
+  @FieldResolver()
+  async difficulty() {
+    try {
+      const response = await this._nodeService.getNodeInfo();
+      return response.data["difficulty"];
+    } catch (e) {
+      if (e instanceof AxiosError) {
+        if (e.response?.data?.error === 400) {
+          throw new GraphQLError(e.response?.data?.detail);
+        }
+      }
+
+      console.error(e);
+      throw new GraphQLError("Unknown error");
+    }
+  }
```

Comment:
If `network` and `difficulty` are selected in the same query, it will require two node roundtrips to the same endpoint. I think we can optimize that by resolving both fields in `state()` using `isFieldSelected()` to assist on selection checking.

This way we can use just one node request to fill both fields, if at least one of them are selected.